docs/monthly-meeting/2023-09.md

```diff
@@ -0,0 +1,118 @@
+Please take a second to read through it!
+
+
+## Roll call
+
+(Name / `@GitHubUsername` / Discord if different)
+- CAM Gerlach / `@CAM-Gerlach`
+- Daniele / `@EvilDMP`
+- Petr Viktorin / `@encukou`
+- Adam Turner / `@AA-Turner`
+- Hugo van Kemenade / `@hugovk`
+- Maciej Olko / `@m-aciek` / `@m_aciek`
+- Ryan / `@ryan-duve`
+- Marcus Sherman / `@betteridiot`
+- Erlend / `@erlend-aasland`
+- Ege Akman / `@egeakman`
+- Ezio Melotti / `@ezio-melotti`
+
+
+## Introductions
+
+- Maciej, from Poland, works on Polish translation
+
+
+## Reports and celebrations
+
+- [Hugo] [docs.python.org](https://docs.python.org) now has a dark theme! 🌒
+
+- [Hugo] Little things can be big things: I added a shortcut at the top of the [datetime](https://docs.python.org/3/library/datetime.html) docs to take you directly to the percent formatting codes. [People are happy!](https://mastodon.social/@hugovk/110951423574144386)
+
+- [Adam] Six releases of [Sphinx](https://www.sphinx-doc.org/en/master/changes.html)!
+
+- [Adam] New PEP helper & validator ([`check-peps.py`](https://github.com/python/peps/commit/814ceede975056cc56a019bbed62b29a6a875830))
+
+- [Adam] [Datamodel headings](https://github.com/python/cpython/pull/108146) for readability
+
+- [Adam] Work to [remove references to Distutils](https://github.com/python/cpython/issues/92584) in the documentation ahead of the 3.12.0 release
+
+
+## Discussion
+
+- [Hugo] python-docs-theme:
+  - Need a release of 2023.8 to fix some dark theme bugs: [python/python-docs-theme#152](https://github.com/python/python-docs-theme/issues/152)
+  - Have a PR ready to release using Trusted Publishers, need someone to set it up on PyPI: [python/python-docs-theme#148](https://github.com/python/python-docs-theme/pull/148)
+  - These are both waiting on people who can make releases. Adam: should we review the group of people who can make releases, and look to expand it? Potentially ties in to a [Python organisation](https://github.com/python/steering-council/issues/207) on PyPI.
+
+
+- [Ege/Hugo/CAM] Updates and discussion on Plausible hosting:
+  - PSF folks opposed to self-hosting
+  - Paid hosting is a *little* expensive
+  - We got a 15% discount from them, which would be \$1400-\$1500 yearly
+  - People brought up several motivating use cases already for this, including:
+    - Translation prioritization for languages/pages
+    - Automated translations
+    - Determine how many people use backported branches
+    - Determining if old pages are still needed/used
+    - Potentially determine common broken links, maybe even fragment ids?
+    - Prioritisation for documentation
+  - [Previous Core Development budget thread](https://discuss.python.org/t/23507)
+  - Next steps: CAM to write to Deb from the PSF with a summary of this discussion
+
+- [Maciej] Translations platform – trying to initiate discussion ([Discourse thread](https://discuss.python.org/t/docs-translation-platform/29940))
```

Comment:
We should capture the outcome of this chat, which was to get in touch with the PSF (with Cam's support)